src/assets/i18n/de_DE.json

```diff
@@ -486,7 +557,7 @@
   "approval.detailspage.cost_center.label": "Kostenstelle",
   "approval.detailspage.costcenter.approval.heading": "Kostenstellen-Genehmigung",
   "approval.detailspage.costcenter.approver.label": "Genehmiger (Kostenstelle)",
-  "approval.detailspage.costcenter.budget.label": "{{0, select, =weekly{Kostenstellenbudget (wöchentlich)} =monthly{Kostenstellenbudget (monatlich)} =quarterly{Kostenstellenbudget (quartalsweise)} =half-yearly{Kostenstellenbudget (halbjährlich)} =yearly{Kostenstellenbudget (jährlich)}}}",
+  "approval.detailspage.costcenter.budget.label": "{{0, select, =weekly{Kostenstellenbudget (wöchentlich)} =monthly{Kostenstellenbudget (monatlich)} =quarterly{Kostenstellenbudget (quartalsweise)} =fixed{Kostenstellenbudget (vordefiniert)}}}",
```

Comment:
Is it correct that there is no half-yearly and yearly here anymore as opposed to key "account.budget.label"?

Reply:
Yes, cost center budgets have different periods than customer budgets.